app/views/idv/review/new.html.erb

```diff
@@ -9,7 +9,7 @@
       ) %>
 <% end %>
 
-<%= render PageHeadingComponent.new.with_content(t('idv.titles.session.review', app_name: APP_NAME)) %>
+<%= render PageHeadingComponent.new.with_content(@title) %>
```

Comment:
Looks like the page title (as opposed to the H1) comes from a different translation key, `titles.idv.review` which currently says 'Re-enter your password'. Do we want to add a new title to match the new H1?